tools/schemacode/bidsschematools/schema.py

```diff
@@ -246,3 +246,35 @@ def filter_schema(schema, **kwargs):
             if isinstance(item, dict):
                 new_schema[i] = filter_schema(item, **kwargs)
     return new_schema
+
+
+def export(output, schema_path=None):
+    """
+    Export BIDS schema to JSON document.
+
+    Parameters
+    ----------
+    output : str
+        A path which to write a JSON exported schema to, or `-` if the output is to be written
+        to stdout.
+    schema_path : str or None, optional
+        A path to the schema directory you wish to export to JSON.
+
+    Examples
+    --------
+    """
+    logger = logging.getLogger("bidsschematools")
+
+    if schema_path:
+        schema_path = os.path.abspath(os.path.expanduser(schema_path))
+
+    schema = load_schema(schema_path)
+    text = _jsonify(schema)
```

Comment:
Let's just get rid of the `export_schema()` function.

```suggestion
    text = schema.to_json()
```

Reply:
ok